### PR TITLE
RefineOp(any) | feat(atenlib)

### PR DESCRIPTION
**Pull request by xiaowuhu**
1. split aten_any() to aten_any_dim()
2. remove trace_only=True
3. remove xfail

### CODE REVIEW

**Review by fatcat-z — APPROVED** [review]
LGTM, thanks!

File: onnxscript/function_libs/torch_lib/ops/core.py [hunk 1]
```diff
@@ -272,31 +272,42 @@ def aten_angle(self: TensorType) -> TensorType:
...
+
+    self_rank = op.Size(op.Shape(self))
+    if self_rank == 0:
+        result = op.Not(op.Equal(self, 0.0))
+    else:
+        # cannot cast to INT64 because 0.1 will be cast to 0, then convert to false
+        self_bool = op.Cast(self, to=BOOL.dtype)
+        # op.ReduceMax() in next step cannot calculate BOOL value, so convert to INT64
+        self_int = op.Cast(self_bool, to=INT64.dtype)
+        # Change int(dim) to tensor(dims)
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_lib/ops/core.py:307]: More precisely is it changing dim from INT64 to INT64[1]?

**Review by justinchuby — APPROVED** [review]

File: onnxscript/function_libs/torch_lib/ops/core.py [hunk 2]
```diff
@@ -272,31 +272,42 @@ def aten_angle(self: TensorType) -> TensorType:
...
-        dims = op.Cast(dim, to=INT64.dtype)
-        result_max = op.ReduceMax(self_int, dims, keepdims=keepdim, noop_with_empty_axes=0)
+        result = op.Not(op.Equal(self, 0.0))
     else:
+        # cannot cast to INT64 because 0.1 will be cast to 0, then convert to false
+        self_bool = op.Cast(self, to=BOOL.dtype)
+        # op.ReduceMax() in next step cannot calculate BOOL value, so convert to INT64
+        self_int = op.Cast(self_bool, to=INT64.dtype)
         result_max = op.ReduceMax(self_int, keepdims=0, noop_with_empty_axes=0)
+        result = op.Greater(result_max, op.Constant(value_int=0))
```
[the remark this answers is not in the document]
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_lib/ops/core.py:291]: It is Greater, not Gather. :)

File: onnxscript/function_libs/torch_lib/ops/core.py (hunk 1 above)
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_lib/ops/core.py]: from int to INT64[1]

File: onnxscript/function_libs/torch_lib/ops/core.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_lib/ops/core.py:291]: It is Greater, not Gather. :)

**Pull request merged**